src/pages/use-cases/code-reuse.tsx

```diff
@@ -166,9 +166,9 @@ const CodeReusePage: FunctionComponent = () => (
             description:
                 'Find existing code libraries for reuse and avoid spending time on problems a teammate already solved for a more secure and coherent codebase.',
         }}
-        className="use-cases-page"
+        className="use-cases-page navbar-light"
```

Comment:
Same here, a sus `navbar-light`

Reply:
Again, good point. Link to our other thread in this PR: https://github.com/sourcegraph/about/pull/5395#discussion_r883100687